server/events/eventMacro.js

```diff
@@ -113,3 +113,10 @@ App.on("triggerMacroButton", ({simulatorId, configId, buttonId}) => {
   if (macros.length > 0)
     App.handleEvent({simulatorId, macros}, "triggerMacros");
 });
+
+// Reorder Macro Action by Sam Hill
```

Comment:
There's no need to leave your signature everywhere - it's already included as part of the git history.